python/tests/test_variable.py

```diff
@@ -110,6 +110,12 @@ def test_create_scalar_quantity():
     assert var.unit == sp.units.m
 
 
+def test_create_via_unit():
+    expected = sp.Variable(1.2, unit=sp.units.m)
+    var = 1.2 * sp.units.m
```

Comment:
Does it also make sense to add the divide operator, if for example you want something like
`var = 14.0 / sp.units.s`
i.e. 14 Hz?

Reply:
Yes I think it would. I mainly added this to try it out. It came up when reviewing https://github.com/scipp/ess/pull/2. If it proves useful we would also want to add it in C++ and have overloads for `int`, etc.